library/hmac_drbg.c

```diff
@@ -62,6 +62,8 @@ void mbedtls_hmac_drbg_init( mbedtls_hmac_drbg_context *ctx )
 {
     memset( ctx, 0, sizeof( mbedtls_hmac_drbg_context ) );
 
+    ctx->prediction_resistance = MBEDTLS_HMAC_DRBG_PR_OFF;
+    ctx->reseed_flag = MBEDTLS_HMAC_DRBG_RESEED;
```

Comment:
I wonder if setting the hamming distance for a value that is never checked makes sense. The `ctx->reseed_flag` could have any value other than MBEDTLS_HMAC_DRBG_NO_RESEED and its behavior will be always the same.

Reply:
The initial value is `MBEDTLS_HMAC_DRBG_RESEED`, and the check is for `!= MBEDTLS_HMAC_DRBG_NO_RESEED` so if a bad guy wants to skip reseeding, he has to change a lot of bits.